content/127labs/drone-k8s-helm/index.md

```diff
@@ -0,0 +1,54 @@
+---
+date: 2017-06-10T00:00:00+00:00
+title: Kubernetes Helm
+author: 127labs
+tags: [ kubernetes, helm, docker ]
+repo: 127labs/drone-k8s-helm
+logo: kubernetes_helm.svg
+image: 127labs/drone-k8s-helm
+---
+
+This Kubernetes plugin can be included into your pipelines to allow deployment with Helm.
+
+Example configuration:
+
+```yaml
+pipeline:
+  ship:
+    image: 127labs/drone-k8s-helm
+    chart: stable/dokuwiki
+    master: https://127.0.0.1:8001
+    token: super-long-token
+    release: wiki
+    skip_tls: true
+    clean_before_release: true
+    values:
+      dokuwikiEmail: $${DOKUWIKI_EMAIL}
+      dokuwikiPassword: $${DOKUWIKI_PASSWORD}
+    secrets: [dokuwiki_email, dokuwiki_password]
+```
+
+# Secret Reference
+
+Values are capable of accessing secrets by interpolating them with the `$${SECRET}` syntax. The plugin will
```

Comment:
What do you think if we use a different syntax instead? (i.e. `{{DOKUWIKI_EMAIL}}`) would it be clearer?

Or do you think it is okay to maintain the syntax but describe in more detail in the documentation?